.github/workflows/markdown-links.yml

```diff
@@ -1,23 +1,27 @@
 name: Check Markdown links
 
-on: [push,  pull_request]
+on: push
 
-jobs:
-  markdown-link-check-push:
-    if: github.event_name == 'push' && github.repository_owner != 'conan-io' # We do not want to see red in CCI
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        config-file: .github/workflows/mlc_config.json
+env:
+  PYVER: "3.8"
 
+jobs:
   markdown-link-check-pr:
     if: github.event_name == 'pull_request'
```

Comment:
This job will never run since this even was removed above 🤔

Reply:
oh, sorry. My mistake!